translator/tocwconfig/sampleConfig/emf_and_kubernetes_config.yaml

```diff
@@ -518,6 +520,7 @@ service:
             processors:
                 - metricstransform/containerinsights
                 - batch/containerinsights
+                - gpu/containerinsights
```

Comment:
Why does it get added here if `"gpu_metrics"` is false in the JSON?  (I think I answered my own question in my next comment.)

Reply:
OK I'm still wondering about this.

Reply:
missed to add test files. added